zfs/zfs.go

```diff
@@ -328,6 +328,10 @@ func (a ZFSSendArgs) buildCommonSendArgs() ([]string, error) {
 		args = append(args, "-w")
 	}
 
+	if a.Properties.B {
+		args = append(args, "-p")
```

Comment:
I guess we're good wrt compatibility with old ZFS versions here (clear from FreeBSD side, I guess ZoL already had this in 0.7, too?)

Reply:
Yup, IRC assured me back then that this is not something new if you're at least on 0.7.x IIRC. As this isn't on by default, I'm against bloating this with a compatibility check.